packages/app/src/cli/services/build/extension.ts

```diff
@@ -91,12 +91,14 @@ export async function buildUIExtension(extension: ExtensionInstance, options: Ex
     env.APP_URL = options.appURL
   }
 
+  const {main, assets} = extension.getBundleExtensionStdinContent()
```

Comment:
I think we should revert this change and not touch the stdin since we no longer need to use it for other assets. We can do a simple map for the build options since we have access to the config

Reply:
Nevermind, it looks like `extension.getBundleExtensionStdinContent` is being called in multiple places including the `createContexts` method in `app-watcher-esbuild` which isn't even using this code path. It would be nice to refactor one day but it doesn't block this from shipping.